webapp/src/pages/UserPage/UserPage.tsx

```diff
@@ -0,0 +1,85 @@
+import './UserPage.css';
+import {
+  Button,
+  ButtonVariant,
+  Header,
+  IconVariant,
+  LargeAvatar,
+  Row,
+  Text,
+  TextColor,
+  TextVariant,
+  TextWeight,
+} from '../../shared/components';
+import { USER_URL, USERS_EP_URL, WILDCARD_AVATAR_URL } from '../../shared/urls';
+import { instanceOfUser, User } from '../../shared/types';
+import { useData } from '../../shared/hooks/UseData';
+import { goBack, logout } from '../../shared/callbacks';
+
+export default function UserPage() {
+  const me = useData<User>(
+    USERS_EP_URL + '/me',
+    {
+      username: 'placeholder-name',
+      email: '',
+      avatarId: null,
+      id: '',
+      createdAt: new Date(Date.now()),
+    },
+    instanceOfUser,
+  );
+
+  return (
+    <div className="user-page">
+      <Header
+        navigationFigure={IconVariant.ARROW_BACK}
+        onClick={goBack}
+        statusVariant="online"
+      >
+        {me.username}
+      </Header>
+      <div className="user-wrapper">
+        {me.id ? (
+          <LargeAvatar
+            url={
+              me.id !== ''
+                ? `${USERS_EP_URL}/${me.id}/avatar`
+                : WILDCARD_AVATAR_URL
+            }
+            caption="level 4"
+          />
+        ) : (
+          <></>
+        )}
+        <div className="user-text">
+          <Text
+            variant={TextVariant.PARAGRAPH} // this size doesn't look like in figma
+            color={TextColor.LIGHT} // at least for desktop
+            weight={TextWeight.MEDIUM}
+          >
+            {me.username}
+          </Text>
+          <Text
+            variant={TextVariant.PARAGRAPH} // this size doesn't look like in figma
```

Comment:
// this size doesn't look like in figma, but it's set to figma's setting

Reply:
Well, I found where is the inconsistency:
![Screenshot from 2022-08-15 14-24-01](https://user-images.githubusercontent.com/2774420/184627405-7b08e234-311f-4ce5-99c2-059e476d0d7d.png)
![Screenshot from 2022-08-15 14-23-47](https://user-images.githubusercontent.com/2774420/184627407-ab55c4a3-969d-4e9c-b523-370cd91b3f36.png)
![Screenshot from 2022-08-15 14-22-37](https://user-images.githubusercontent.com/2774420/184627411-9d6f8aa1-bdae-46bf-bfc2-33ecfe7cb534.png)

For desktop, for our src code as it is: font-size is set to 18, paragraph is set to 1rem, which means that paragraph is set to 18 font size.
For figma, caption is set to 18, while caption is set to 21.

As a solution, I have to say that I think font size should stay at 18 for desktop, and it should stay in paragraph. Figma paragraph should match 18 font size for paragraph, and we should rework figma layouts where it doesn't look okay: change some of the texts from paragraph to subheading, and so on. I think this missmatch is exclusive to desktop.